src/base/lnav_log.cc

```diff
@@ -274,7 +274,7 @@ void log_msg(lnav_log_level_t level, const char *src_file, int line_number,
         localtm.tm_sec,
         (int)(curr_time.tv_usec / 1000),
         LEVEL_NAMES[to_underlying(level)],
-        basename((char *)src_file),
+        basename((char *)::strdup(src_file)),
```

Comment:
This will be called for every log message, that seems like a perf issue (and a leak since it's not freed?).  There's a `sql_basename` in fs-extension-funcs.cc.  Maybe move that somewhere else and use it here?

Reply:
Sorry. Never realized that this gets called for every log message. I was just trying to fix the segfault during doc generation at compile time.

Will fix it.